src/Makefile

```diff
@@ -140,6 +141,13 @@ $(SUBDIRS) : mklibdir
 $(EXT_SUBDIRS) : mklibdir
 	$(MAKE) -C $@
 
+install: all
+	mkdir -p $(INSTALLPREFIX)/{bin,lib}
+	# copying binaries to $(INSTALLPREFIX)/bin
+	find $(SUBDIRS_BIN) -type f -executable -exec cp -v {} $(INSTALLPREFIX)/bin/ ';'
+	# copying libs to $(INSTALLPREFIX)/lib
+	cp -av ../tools/openfst/lib/* $(INSTALLPREFIX)/lib/
+
```

Comment:
Is there a particular reason why you install the openfst libst, but not binaries?
I think some of them are used by the scripts in the egs/
I'm fairly indecisive what would be the correct thing to do, especially given the fact that the openfst we are using is patched.